conversion-measurement/README.md

```diff
@@ -5,17 +5,17 @@
 ## Set and run locally
 
 1. Run `git clone https://github.com/GoogleChromeLabs/trust-safety-demo.git && cd trust-safety-demo/conversion-measurement`
-1. Make sure you have [node.js](https://nodejs.org/en/download/) and [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) installed.
-1. Run `sh ./init.sh` to download and install all dependencies.
-1. Run `sh ./start.sh` to startup several local node apps.
-1. You now have multiple servers running: home(:8080), publisher(:8081), advertiser(:8082), adtech(:8084) server.
+1. Make sure you have [node.js](https://nodejs.org/en/download/) and the latest Firebase CLI `npm install -g firebase-tools` installed.
+1. Install all dependencies in the functions folder `cd functions && npm install && cd ..`.
+1. Run the Firebase emulator with `firebase emulators:start` to startup all the sites.
+1. You now have multiple servers running: home(:8080), adtech(:8085), advertiser(:8086), publisher(:8087) server.
 1. Open [localhost:8080](http://localhost:8080) in Chrome.
```

Comment:
Question for my understanding:
Are we changing this to avoid colliding with common port numbers?
If so, shall we also change the 8080 one?

Reply:
That are the default ports Firebase emulators is using, I have not found a way to pin any other than the first one (`8080`). I have added some more info and troubleshooting guidance around this.

Reply:
Understood ty, nit: Do you think we could add a comment above `__session=` to mention this, as to not confuse folks at to why this cookie is called session?